typesetters/base.lua

```diff
@@ -944,12 +1135,28 @@ function typesetter:breakpointsToLines (breakpoints)
     if point.position ~= 0 then
       local slice = {}
       local seenNonDiscardable = false
+      local seenLiner = false
+      local lastContentNodeIndex
```

Comment:
poorly name: it's were the "actual" content starts (past zero box, non-explicit glues etc.) but its thus a _first_ content index in the line. Or event better, the `startContentNodeIndex`

Did I ever tell I always confuse left/right, east/west, etc. :D

Reply:
See above, kept, but then it must be updated after repeating entry liner nodes (= `lastContentNodeIndex = #slice + 1` after the call to `_repeatEnterLiners` around line 1134